client/sites/v2/utils/router.ts

```diff
@@ -5,10 +5,15 @@ import type { AnyRoute, AnyRouter } from '@tanstack/react-router';
 
 export function getRouterOptions() {
 	return {
+		context: {
+			auth: undefined,
+			config: undefined,
+			previousLocationRef: undefined,
+		},
 		defaultPreload: 'intent' as const,
 		defaultPreloadStaleTime: 0,
 		defaultNotFoundComponent: () => null,
-		defaultViewTransition: true,
+		defaultViewTransition: false,
```

Comment:
Disabled the transition because the UI appears odd. The main reason is that the header is hidden on the overview page. With the transition enabled, the overview page’s header briefly appears, which can be confusing for users.